crates/bevy_render/src/texture/image.rs

```diff
@@ -256,17 +256,34 @@ impl Image {
 
     /// Returns the aspect ratio (height/width) of a 2D image.
     pub fn aspect_2d(&self) -> f32 {
```

Comment:
I would honestly like to see this method renamed to `aspect_ratio` while we're here.